ios/brave-ios/Sources/BraveShields/ShieldStrings.swift

```diff
@@ -321,7 +321,8 @@ extension Strings.Shields {
     "EnableGPCDescription",
     tableName: "BraveShared",
     bundle: .module,
-    value: "Enable the Global Privacy Control JS API",
+    value:
+      "Ask websites not to sell or share your data. [Learn more](https://support.brave.com/hc/en-us/articles/360017989132-How-do-I-change-my-Privacy-Settings#h_01HHFRRT6B5YTRREA4ZDARGTWM)",
```

Comment:
@ShivanKaul We may want to update the iOS paragraph in the support article to align closer to the new copy, maybe after this lands in Release?

Reply:
Yeah, I included that in the issue description.